missions/W4/M1/Dockerfile

```
@@ -0,0 +1,89 @@
+# ubuntu-20.04 base.
+FROM ket0825/spark-base:latest
+
+# 필수 패키지 설치
+# DEBIAN_FRONTEND=noninteractive: apt-get install 시 interactive 설정을 무시하고 설치 진행 (모두 yes로 설정)
+RUN sudo apt-get update -y \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)